packages/@aws-cdk-testing/cli-integ/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aws-cdk-testing/cli-integ",
   "description": "Integration tests for the AWS CDK CLI",
-  "version": "0.0.0",
+  "version": "2.96.2",
```

Comment:
Anything we have listed as version `0.0.0` should not be changed (also in the instances below).